### PR TITLE
Add OpenVINO regression tests

### PR DESCRIPTION
**Pull request by l-bat**
### Changes

* Added POT and OpenVINO Native regression tests.

### Reason for changes

* To control the quality of OpenVINO backends.

### Related tickets

* 103898

### Tests
* `tests/openvino/pot/quantization/test_sanity.py`
*  `tests/openvino/native/quantization/test_sanity.py`

### CODE REVIEW

File: tests/onnx/benchmarking/dataset_definitions.yml [hunk 1]
```diff
@@ -1495,3 +1495,17 @@ datasets:
       annotation_file: object_detection/streams_1/high/annotations/instances_glb2bcls3.json
     annotation: mscoco_detection_high_3cls.pickle
     dataset_meta: mscoco_detection_high_3cls.json
+
+  - name: imagenette2-320
```
Comment (alexsu52) [inline tests/onnx/benchmarking/dataset_definitions.yml:1499]: Using any files in OpenVINO backend from ONNX backend is not good practice. Could you move `dataset_definition.yml` to a share part of the code, for example in the `tests/shared/data`?

File: tests/openvino/omz_helpers.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        "-d", str(DATASET_DEFINITIONS_PATH),
+        "-s", str(data_dir),
+        "-tf", framework,
+        "-td", device,
+        "--csv_result", str(report_path)
+    ]
+    if eval_size is not None:
+        com_line += ["-ss", str(eval_size)]
+
+    run_command(com_line)
```
Comment (alexsu52) [inline tests/openvino/omz_helpers.py:93]: You can use the shared code here https://github.com/openvinotoolkit/nncf/blob/develop/tests/shared/command.py

**Review by alexsu52 — CHANGES_REQUESTED** [review]

**Review by alexsu52 — COMMENTED** [review]
Some TODO from my side:
- Add `--data` option to specify the path to the dateset cache
- Add the `--data` option to the `test-openvino` target from the makefile
- Create a ticket to support the `--data` option on the CI side

File: tests/onnx/benchmarking/dataset_definitions.yml (hunk 1 above)
Comment (l-bat) [reply tests/onnx/benchmarking/dataset_definitions.yml]: Created specific `ov_dataset_definition.yml` for  OpenVINO backend

File: tests/openvino/omz_helpers.py (hunk 2 above)
Comment (l-bat) [reply tests/openvino/omz_helpers.py:75]: Done

**Comment by l-bat** [comment]
Ticket to support the `--data` option on the CI side: 104815

**Review by alexsu52 — APPROVED** [review]
LGTM

**Pull request merged**